Source/EasyNetQ.Examples.DlxRetry/EntryPoint.cs

```diff
@@ -10,10 +10,10 @@
 
 var serviceCollection = new ServiceCollection();
 
-serviceCollection.AddEasyNetQ("host=localhost");
 serviceCollection.AddLogging(builder => builder.AddConsole());
-serviceCollection.EnableNewtonsoftJson();
-serviceCollection.EnableAlwaysAckConsumerErrorStrategy();
+serviceCollection.AddEasyNetQ("host=localhost")
```

Comment:
Maybe there can be an overload of AddEasyNetQ that accepts Action<IEasyNetQBuilder>

Reply:
Do you mean instead of returning IEasyNetQBuilder it might be better to accept a delegate of type Action<IEasyNetQBuilder> in AddEasyNetQ method?

Reply:
maybe both?